### PR TITLE
When clicking on a tab, the first field now has the focus

### PR DESCRIPTION
**Pull request by Braunch**
- [x] Change in CHANGELOG.md described?
- [x] Changes in pull request outlined? (What, why, ...)
- [x] Tests created for changes?
- [x] Tests green?

This is referes to #623 
I just had to add the activateVisible() Method to the ChangeListener to ensure the focus is set.

### CODE REVIEW

**Comment by simonharrer** [comment]
What happens if I am scrolling through the list of entries in the table via the keys? Does the focus can jump away to the field editor if it is open? From what I recall, we had the issue I mentioned before and had to remove the functionality to automatically focus the first field. Please test this exhaustively with different entry editors for different entry types open and then scrolling via keys.

**Comment by Braunch** [comment]
I am going to figure that out after our exam this week. So your comment is recognized :)

**Comment by Braunch** [comment]
I had a chat with @koppor  today. @bruehldev  and I will write an automated test for this in our upcoming sprint (starting on monday 25th april). When this is done we should have verified if the focus is lost on scrolling or not.

**Comment by stefan-kolb** [comment]
What is the current status here?

**Comment by Braunch** [comment]
I have run the Integration tests on it and tried it manually. The entry tables focus is not lost when going through the entries. Only when opening the entry editor as intended.

**Comment by simonharrer** [comment]
I wonder why the travis jobs are not executed? Because you violate the correct imports order, and this cannot be merged without the build failing at the moment.

**Comment by Braunch** [comment]
I will fix the order soon. Probably on Friday. Maybe pushing on the branch will then trigger the jobs again.

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - When resolving duplicate BibTeX-keys there is now an "Ignore" button. "Cancel" and close key now quits the resolving.
 - The [online forum](http://discourse.jabref.org/) is now directly accessible via the "Help" menu
 - Implemented [#1338](https://github.com/JabRef/jabref/issues/1338): clicking on a crossref in the main table selects the parent entry and added a button in the entry editor to select the parent entry.
+- [#623](https://github.com/JabRef/jabref/issues/623) The first Field does now have focus when clicking on a tab in the entry editor
```
Comment (tobiasdiez) [inline CHANGELOG.md]: The first field now has focus ...

File: CHANGELOG.md [hunk 2]
```diff
@@ -183,6 +184,7 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - Swedish is added as a language option (still not a complete translation)
 - [#969](https://github.com/JabRef/jabref/issues/969) Adding and replacing old event system mechanisms with Google Guava EventBus.
 
+
```
Comment (tobiasdiez) [inline CHANGELOG.md]: I think this is new line happened by accident.

File: CHANGELOG.md [hunk 3]
```diff
@@ -433,4 +435,4 @@ The changelog of 2.11 and versions before is maintained as [text file](https://g
 [3.1]: https://github.com/JabRef/jabref/compare/v3.0...v3.1
 [3.0]: https://github.com/JabRef/jabref/compare/v2.11.1...v3.0
 [dev_2.11]: https://github.com/JabRef/jabref/compare/v2.11.1...dev_2.11
-[2.11.1]: https://github.com/JabRef/jabref/compare/v2.11...v2.11.1
+[2.11.1]: https://github.com/JabRef/jabref/compare/v2.11...v2.11.1
```
Comment (tobiasdiez) [inline CHANGELOG.md]: String "diff"

**Comment by tobiasdiez** [comment]
LGTM :+1:, just fix the import order (and improve the changelog), then this can be merged in from my point of view

**Comment by Braunch** [comment]
Can this be merged?

**Comment by simonharrer** [comment]
why did the gradlew scripts change?

**Comment by simonharrer** [comment]
otherwise LGTM

**Comment by Braunch** [comment]
Gradle scripts are back to normal.

**Comment by stefan-kolb** [comment]
OK :+1:

**Pull request merged**